src/zero/util.clj

```diff
@@ -68,8 +68,8 @@ vault.client.http/http-client           ; Keep :clint eastwood quiet.
                       .getTokenValue)]
     (when-not token
       (let [lines ["%1$s: Cannot generate token from Google Credentials."
-                   "%1$s: Run 'gsutil auth list' to check your account."
-                   "%1$s: Run 'gsutil auth login' and try again."]
+                   "%1$s: Run 'gsutil auth list' to check your account if running locally"
+                   "%1$s: Try to login if running on a server."]
```

Comment:
How will you see this on a server?

Reply:
🤔 Good question... You won't ... we need to make the error handling better and be able to pass the messages all the way from API to UI.